org.theia_ide.theia_ide.yaml

```diff
@@ -0,0 +1,30 @@
+app-id: org.theia_ide.theia_ide
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node20
+modules:
+  - shared-modules/libsecret/libsecret.json
+  - theia_ide.yaml
+command: run.sh
+finish-args:
+  - --allow=devel
+  - --device=dri
+  - --env=ELECTRON_TRASH=gio
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --filesystem=home
+  - --filesystem=host-os:ro
+  - --filesystem=xdg-cache
+  - --filesystem=xdg-config
+  - --filesystem=xdg-data
```

Comment:
why does it need access to this xdg- directories?

Reply:
Theia can access the home directory. If the user set, say, `XDG_CONFIG_HOME`, outside their home, wouldn't Theia need access to xdg-config to stay consistent?